### PR TITLE
DOC: Try to improve find_bad_channels_maxwell doc

### PR DESCRIPTION
**Pull request by hoechenberger**
#### What does this implement/fix?
Try to make the algorithm description more accessible.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@larsoner If I'm not mistaken, we're calculating **SD** to find flat channels, not **variance** as stated in the comments. I've changed this in 94a51ca.

**Comment by hoechenberger** [comment]
rendered documentation:
https://21024-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.find_bad_channels_maxwell.html?highlight=find_bad_channels_maxwell#mne.preprocessing.find_bad_channels_maxwell

File: mne/preprocessing/maxwell.py [hunk 1]
```diff
@@ -1923,15 +1931,19 @@ def find_bad_channels_maxwell(
...
         - ``scores_flat`` : ndarray, shape (n_meg, n_windows)
-            The scores for testing whether MEG channels are flat.
+            The scores for testing whether MEG channels are flat. These values
+            correspond to the standard deviation of a segment.
+            See the ``Notes`` section for details.
         - ``limits_flat`` : ndarray, shape (n_meg, 1)
-            The score thresholds above which a segment was claffified as
-            "flat".
+            The score thresholds (in standard deviation) above which a segment
+            was claffified as "flat".
```
Comment (agramfort) [inline mne/preprocessing/maxwell.py:1939]:
```suggestion
            was classified as "flat".
```

File: mne/preprocessing/maxwell.py [hunk 2]
```diff
@@ -1923,15 +1931,19 @@ def find_bad_channels_maxwell(
...
         - ``scores_noisy`` : ndarray, shape (n_meg, n_windows)
-            The scores for testing whether MEG channels are noisy.
+            The scores for testing whether MEG channels are noisy. These values
+            correspond to the standard score of a segment.
+            See the ``Notes`` section for details.
         - ``limits_noisy`` : ndarray, shape (n_meg, 1)
-            The score thresholds above which a segment was claffified as
-            "noisy".
+            The score thresholds (in standard scores) above which a segment was
+            claffified as "noisy".
```
Comment (agramfort) [inline mne/preprocessing/maxwell.py:1946]:
```suggestion
            classified as "noisy".
```

Comment (hoechenberger) [reply mne/preprocessing/maxwell.py:1946]: Slight lips there LOL :)

**Comment by larsoner** [comment]
Thanks @hoechenberger

**Pull request merged**